test/bondingEvent.js

```diff
@@ -332,90 +339,129 @@ describe('BondingEvent', async () => {
         const expectedLiquidity = initialLiquidityTotal.mul(2);
         expect(position.liquidity).to.equal(expectedLiquidity);
       });
+
+      it('creates a second position if the price moves after initialisation', async () => {
+        // will create the first position
+        let amountSEuro = etherBalances.TWO_MILLION;
+        let { amountOther } = await BondingEvent.getOtherAmount(amountSEuro);
+        await SEuro.connect(customer).approve(BondingEvent.address, amountSEuro);
+        await USDT.connect(customer).approve(BondingEvent.address, amountOther);
+        await BondingEvent.connect(owner).bond(
+          customer.address, amountSEuro, durations.ONE_YR_WEEKS, rates.TEN_PC,
+        );
+
+        let positions = await BondingEvent.getPositions();
+        expect(positions).to.be.length(1);
+        let position = await BondingEvent.getPositionData(positions[0]);
+        expect(position.lowerTick).to.equal(pricing.lowerTick);
+        expect(position.upperTick).to.equal(pricing.upperTick);
+        expect(position.liquidity).to.be.gt(0);
```

Comment:
I see, it's a reasonable point. we shouldn't get too deep in testing uniswap's mechanics too, as I've mentioned before, so it's fine. usually if it's possible, it would be nice to know what to expect.